spec/scenic/adapters/postgres_spec.rb

```diff
@@ -51,16 +59,44 @@ module Adapters
 
           expect(adapter.views.map(&:name)).not_to include("greetings")
         end
+
+        it "raises an exception if the version of PostgreSQL is too old" do
+          adapter = Postgres.new
+          allow(adapter).to receive(:version_number) { 920 }
+
+          expect { adapter.drop_materalized_view("greetings") }
+            .to raise_error StandardError
+        end
       end
 
       describe "#refresh_materialized_view" do
-        it "raises descriptive error if concurrent refresh is not possible" do
+        it "raises an exception if the version of PostgreSQL is too old" do
           adapter = Postgres.new
-          adapter.create_materialized_view(:tests, "SELECT text 'hi' as text")
+          allow(adapter).to receive(:version_number) { 920 }
 
           expect {
-            adapter.refresh_materialized_view(:tests, concurrently: true)
-          }.to raise_error(/Create a unique index with no WHERE clause/)
+            adapter.refresh_materialized_view(:tests)
+          }.to raise_error StandardError
+        end
+
+        context "refreshing concurrently" do
+          it "raises descriptive error if concurrent refresh is not possible" do
+            adapter = Postgres.new
+            adapter.create_materialized_view(:tests, "SELECT text 'hi' as text")
+
+            expect {
```

Comment:
Feel free to ignore these, as I'm pretty sure it matches what @derekprior has been doing elsewhere.